src/functions/show.ts

```diff
@@ -0,0 +1,12 @@
+import { SelectFields, selectUsers } from "../modules/db"
+import { User } from "../types"
+
+
+
+export const searchUsers = 
```

Comment:
This function is just a wrapper for the `selectUsers` function.
It makes sense to have it like this if we want to extend it in the future, but, for now I believe this can be simplified to
`export const searchUsers = selectUsers`

Reply:
I usually try to strive to extensibility, especially in such a function which would very easily be a target for it. I can change it, but it really would not be my ideal solution.